### PR TITLE
Add: script for `update flake lock`

### PR DESCRIPTION
**Pull request by a-kenji**
##### Description

Take commands out of the `action.yml` file, and put it in a dedicated
shell script. I worked with actions recently and figured out that `$GITHUB_ACTION_PATH` refers to the root of the action runner, and remembered that taking the commands out of the `yaml` was a goal of this project.

##### Checklist

- [x] Tested functionality against a test repository (see ["How to test changes"](../README.md#how-to-test-changes))
- [ ] Added or updated relevant documentation (leave unchecked if not applicable)

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -24,22 +24,16 @@ inputs:
...
-        fi
+    - run: $GITHUB_ACTION_PATH/update-flake-lock.sh
       shell: bash
       env:
         GIT_AUTHOR_NAME: github-actions[bot]
         GIT_AUTHOR_EMAIL: <github-actions[bot]@users.noreply.github.com>
         GIT_COMMITTER_NAME: github-actions[bot]
         GIT_COMMITTER_EMAIL: <github-actions[bot]@users.noreply.github.com>
+        TARGETS: ${{ inputs.inputs }}
+        COMMIT_MSG: ${{inputs.commit-msg}}
```
Comment (cole-h) [inline action.yml:35]: Style nit:
```suggestion
        COMMIT_MSG: ${{ inputs.commit-msg }}
```

**Review by cole-h — COMMENTED** [review]
Oh, I had no idea that env var existed! I'll take a closer look at this tomorrow, but thanks for opening this PR!

**Comment by a-kenji** [comment]
Awesome, no rush! Yeah, I just stumbled across the variable by accident, more or less.

**Review by cole-h — APPROVED** [review]
Thanks, looks good! I pushed a commit re-adding our shellcheck CI and an action that will use ourselves to keep our own flake.lock up to date :)

Thanks a ton for reading about this env var and remembering us :heart:

**Pull request merged**